### PR TITLE
Address todos and deprecations for 2.0

### PR DESCRIPTION
**Pull request by blink1073**
## References
Partially addresses #7720 

## Code changes
Remove deprecations and implement some TODOs for 2.0.

## User-facing changes
None

## Backwards-incompatible changes
Some deprecation warnings are implemented.  The signature of our `commands` functions has been simplified.

### CODE REVIEW

**Comment by blink1073** [comment]
@jasongrout, are you able to review this one?

**Comment by blink1073** [comment]
@telamonian, same question.  I'd like to include this in the beta today.

**Comment by jasongrout** [comment]
I can review this tomorrow, but not today.

**Comment by jasongrout** [comment]
(I'm happy to do the beta release tomorrow as well, if it turns out that I need to review this tomorrow)

File: packages/notebook/src/panel.ts [hunk 1]
```diff
@@ -157,16 +146,6 @@ export class NotebookPanel extends DocumentWidget<Notebook, INotebookModel> {
     super.dispose();
   }
 
-  /**
-   * Handle `'activate-request'` messages.
-   */
-  protected onActivateRequest(msg: Message): void {
-    super.onActivateRequest(msg);
-
-    // TODO: do we still need to emit this signal? Who is using it?
```
Comment (telamonian) [inline packages/notebook/src/panel.ts]: I assume this means that there really was nothing consuming the `activated` signal, right?

Comment (blink1073) [reply packages/notebook/src/panel.ts]: Correct, thanks for the review!

**Pull request merged**